packages/components/menu/src/menu.ts

```diff
@@ -116,7 +116,7 @@ export default defineComponent({
     const activeIndex = ref<MenuProvider['activeIndex']>(props.defaultActive)
     const items = ref<MenuProvider['items']>({})
     const subMenus = ref<MenuProvider['subMenus']>({})
-
+    const activeItemFirstOpend = ref<boolean>(true)
```

Comment:
It seems that there is no need to use `ref`
```suggestion
    let activeItemFirstOpend = true
```